pkg/start/start.go

```diff
@@ -291,7 +299,7 @@ func (o *Options) NewControllerContext(cb *ClientBuilder) *Context {
 			resyncPeriod(o.ResyncInterval)(),
 			cvInformer.Config().V1().ClusterVersions(),
 			sharedInformers.Config().V1().ClusterOperators(),
-			cb.RestConfig(),
+			cb.RestConfig(increaseQPS),
```

Comment:
this will only affect the generic resource builders. what about the `ClientOrDie, KubeClientOrDie` atleast KubeClientOrDie ?

Reply:
I left those at the defaults, those have low qps and low actions (rest config is what is used for sync loop).  I'd like to do a shared qps pool eventually though for everything that isn't status updates to CV (which is effectively what this PR is).